### DISCUSSION TITLE
Search or filter large NDJSON files

### DISCUSSION THREAD

**Discussion by collimarco** [simdjson/simdjson#2023] [General]
I have made some benchmarks and this library is great.

I was wondering if it also supports some sort of filtering to load only the lines that match specific conditions: a sort of basic filter on the JSON lines. For example if you want to find some log lines that contain status=500 or a keyword.

Does this project supports something like this? Otherwise, do you know of any libraries (maybe built on this one) that allow that?

**Comment by lemire** [comment]
You can definitively implement something of the sort with simdjson.
